acceptance/tests/resource/group/should_manage_attributes_aix.rb

```diff
@@ -0,0 +1,24 @@
+test_name "should correctly manage the attributes property for the Group (AIX only)" do
+  confine :to, :platform => /aix/
+  
+  tag 'audit:medium',
+      'audit:refactor',  # Use block style `test_run`
```

Comment:
Why didn't you change these to block style per the refactor comment?

Reply:
I think it is in block style -- just forgot to delete that comment when I copy-pasted an old test file. But definitely worth bringing it up during PR review. Thanks for catching this!

Reply:
Need to remove `audit:refactor`.